common/src/main/java/dev/ftb/packcompanion/features/ServerFeature.java

```diff
@@ -0,0 +1,31 @@
+package dev.ftb.packcompanion.features;
```

Comment:
Feels like this class could be a subclass of CommonFeature to reduce duplication, but not a big deal